Client/Frontend/Browser/HomePanel/NTPModels.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+protocol NTPBackgroundProtocol {
```

Comment:
I went for flexibility because in the future the super referrer may want us to add custom colors for the app for example.
Top sites don't belong to NTP SI, it is a part of NTP in general so even now we start to moving beyond changing the ntp images only